source/docs/hardware/hardware-basics/robot-battery.rst

```diff
@@ -0,0 +1,62 @@
+.. include:: <isonum.txt>
+
+Robot Battery
+=============
+
+The power supply for an FRC\ |reg| robot is a single 12V 18Ah battery. The batteries used for FRC are sealed lead acid batteries capable of meeting the high current demands of an FRC robot.
+
+Battery Connections
+^^^^^^^^^^^^^^^^^^^
+
+Great care should be taken when handling batteries.  **DO NOT CARRY A BATTERY BY THE CABLES**.  The battery will not be able to provide the same amount of current if the :ref:`connectors are loose <docs/hardware/hardware-basics/preemptive-troubleshooting:Check Battery Connections>`.
+
+.. image:: images/robot-battery/heatshrink.png
+   :alt: One terminal of an FRC battery fully covered in heatshrink.
+
+It is a good idea to secure the battery with either a star washer or a nord lock washer as this will ensure the connection does not move.  This connection must also be wrapped, while some use electrical tape heatshrink that fits over the entire connection is recommended.
+
+Battery Longevity
+^^^^^^^^^^^^^^^^^
+
+This battery is rated for about 1200 charge/recharge cycles, but for the high currents FRC subjects it to that lifespan is reduced to about 400 cycles.  These cycles are intended to be relatively limited discharge and deep cycling the battery (running it all the way down) can damage it.
+
+Batteries last the longest if they are kept fully charged when not it use.  Take care however to not leave them on a trickle charger for an extended period of time.
+
+Dropping a battery may compromise its structural integrity and can lead to a bulge or crack in the battery case.  These batteries should be safely disposed of as soon as possible.
+
+Battery Load Tester
+^^^^^^^^^^^^^^^^^^^
+
+.. image:: images/robot-battery/load-tester.jpg
+   :alt: Output screen of a common load tester showing properties of the battery.
+
+A battery load tester can be used as a quick way to determine the readiness of a battery.  It may provide information like: open-load voltage, voltage under load, internal resistance, and state of charge.  These metrics can be used to quickly confirm that a battery is ready for a match and even possibly help to identify some long term problems with the battery.
+
+While these batteries are called 12 volt batteries a fully charged battery should read around 13 volts.  Ideal internal resistance should be around 0.015 Ohms.  If a battery gets higher than 0.02 Ohms it is a good idea to consider not using that battery for competition matches.
```

Comment:
```suggestion
While these batteries are called 12 volt batteries, a fully charged battery should read around ~12-13 volts.  Ideal internal resistance should be around 0.015 Ohms.  If a battery gets higher than 0.02 Ohms it is a good idea to consider not using that battery for competition matches.
```

Perhaps make this a range, as I have seen brand new batteries read 12.1-13.2.

Reply:
I don't think a brand new battery should read 12.1.  My point was intended to be that the voltage should be much closer to 13 than 12.  If others think this is too narrow I can change but IMO a new battery that doesn't charge to more than 12.5 should be taken back.